.meta/transforms/dedupe.toml

```diff
@@ -0,0 +1,41 @@
+[transforms.dedupe]
+title = "Dedupe events"
+allow_you_to_description = "prevent duplicate Events from being outputted by using an LRU cache"
+beta = false
+common = false
+function_category = "filter"
+input_types = ["log"]
+output_types = ["log"]
+requirements = {}
+
+<%= render("_partials/_component_options.toml", type: "transform", name: "dedupe") %>
+
+[transforms.dedupe.options.fields]
+type = "table"
+description = "Options controlling what fields to match against"
+required = true
+
+[transforms.dedupe.options.fields.children.match]
+type = "[string]"
+common = true
+examples = [["field1", "field2"]]
```

Comment:
Mind adding a `parent.child` example? This should demonstrate that nested fields are supported.

Reply:
Done